src/game-engine/config/index.ts

```diff
@@ -20,6 +20,11 @@ import { ItemSpawn, loadItemSpawnConfigurations } from '@engine/config/item-spaw
 import { loadSkillGuideConfigurations, SkillGuide } from '@engine/config/skill-guide-config';
 import { loadMusicRegionConfigurations, MusicTrack } from '@engine/config/music-regions-config';
 import { loadXteaRegionFiles, XteaRegion } from '@runejs/filestore';
+import {
```

Comment:
Prefer to find a way to load these without bringing plugin-specific info into the game engine. Generally we should only pass things to plugins, never import from plugins to the game engine.

Reply:
That's a good point. I do notice the difference now between the other methods. I can definitely move that somewhere more appropriate.